zerver/openapi/zulip.yaml

```diff
@@ -5677,17 +5677,22 @@ paths:
                 type:
                   description: |
                     The type of scheduled message to be sent. `"direct"` for a direct
-                    message and `"stream"` for a stream message.
+                    message and `"stream"` or `"channel"` for a stream message.
 
                     In Zulip 7.0 (feature level 174), `"direct"` was added as the
                     preferred way to indicate the type of a direct message, deprecating
                     the original `"private"`. While `"private"` is supported for
                     scheduling direct messages, clients are encouraged to use to the
                     modern convention because support for `"private"` may eventually
                     be removed.
+
+                    **Changes**: In Zulip 9.0 (feature level 248), `"channel"` was added as
+                    an additional value for this parameter to indicate the type of a stream
+                    message.
```

Comment:
Hmm, should this be above the 7.0 item? I'm not sure why it didn't have a "Changes" heading, but guessing that's a typo, both here and in the other scheduled messages entry.

Reply:
It wasn't a **Changes** note because these endpoints were added after that feature level. The type was changed in feature level 174 and the endpoint was added in 179.

I think we can drop the first sentence and just leave the bit about "private" being deprecated.

Reply:
OK.